package.json

```diff
@@ -10,6 +10,7 @@
     "@testing-library/user-event": "^12.1.10",
     "@toast-ui/editor": "^2.5.1",
     "@toast-ui/react-editor": "^2.5.1",
+    "@toast-ui/toastmark": "0.0.1-alpha.1",
```

Comment:
this dependency comes with the caveat: 
> Currently, ToastMark is for interal usage only as the API's are supposed to be changed frequently. We are planning to register this as a separate npm package when API's are stabilized.

it hasn't changed in 3 months so we're good, right? ;) more seriously, our dependency is quite small; we should be able to update our usage to any API changes or even transition to a different md parser (hopefully not!) in a matter of hours. i also think the aspects of the API that are liable to change are more related to their incremental parsing functionality, not the AST.

that said, we should make the tui.editor maintainers aware of our package & use of `toastmark` after we publish.

Reply:
Ah that makes sense! For this use case, we should be good with this version of Toast, correct? Even if they update etc, what's stopping us from "freezing" this versions of Toast/Toastmark?